src/Mvc/Mvc.ApiExplorer/test/EndpointMetadataApiDescriptionProviderTest.cs

```diff
@@ -610,43 +610,14 @@ public void HandleAcceptsMetadata()
                 context.Results.SelectMany(r => r.SupportedRequestFormats),
                 requestType =>
                 {
-                    Assert.Equal("application/json" , requestType.MediaType);
+                    Assert.Equal("application/json", requestType.MediaType);
                 },
                 requestType =>
                 {
                     Assert.Equal("application/xml", requestType.MediaType);
                 });
         }
 
-        [Fact]
-        public void HandleAcceptsMetadataWhenEmptyReturnsDefaultContentType()
-        {
-            // Arrange
-            var builder = new TestEndpointRouteBuilder(new ApplicationBuilder(null));
-            builder.MapPost("/api/todos", () => "")
-                .Accepts(typeof(string), "");
-            var context = new ApiDescriptionProviderContext(Array.Empty<ActionDescriptor>());
-
-            var endpointDataSource = builder.DataSources.OfType<EndpointDataSource>().Single();
-            var hostEnvironment = new HostEnvironment
-            {
-                ApplicationName = nameof(EndpointMetadataApiDescriptionProviderTest)
-            };
-            var provider = new EndpointMetadataApiDescriptionProvider(endpointDataSource, hostEnvironment, new ServiceProviderIsService());
-
-            // Act
-            provider.OnProvidersExecuting(context);
-            provider.OnProvidersExecuted(context);
-
-            // Assert
-            Assert.Collection(
-                context.Results.SelectMany(r => r.SupportedRequestFormats),
-                requestType =>
-                {
-                    Assert.Equal("application/json", requestType.MediaType);
```

Comment:
What's the behavior now? Is the `MediaType` just `""` or something? Let's just add a regression test for whatever we think the behavior should be in this case rather than delete the test.